doc/translat.xml

```diff
@@ -0,0 +1,459 @@
+    <Description>
+        All and only those translations [bitranslations] of a normalised RMS 
+        created through their tuple [triple] representation lie in these 
+        categories.
+        <C>IsLeftTranslationOfNormalRMS</C> and
+        <C>IsRightTranslationOfNormalRMS</C> are subcategories 
+        of <C>IsTranslationOfNormalRMS</C>, which is a subcategory of 
+        <C>IsTranslationsSemigroupElement</C>.
+        <C>IsBitranslationOfNormalRMS</C> is a subcategory of 
+        <C>IsBitranslation</C>.
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="XTranslationOfNormalRMS">
+  <ManSection>
+    <Func Name="LeftTranslationOfNormalRMS" Arg="L, gpfunc, t"/>
+    <Func Name="RightTranslationOfNormalRMS" Arg="R, gpfunc, t"/>
+    <Returns>A left or right translation.</Returns>
+    <Description>
+      For a function from the rows [columns] of the underlying semigroup
+      (a normalised Rees matrix semigroup) of L [R] to the underlying group of
+      that semigroup, and a transformation on the rows [columns] of the 
+      semigroup, <C>LeftTranslationOfNormalRMS</C>(<A>L, gpfunc, t</A>) 
+      [<C>RightTranslationOfNormalRMS</C>(<A>R, gpfunc, t</A>) returns
+      the corresponding translation as defined in <Cite Key="Clifford1977aa"/>. 
+      <Example><![CDATA[
+gap> G := Group([(1, 2, 3, 5)(4, 10, 7, 12)(6, 11, 9, 8),
+> (1, 3)(2, 5)(4, 7)(6, 9)(8, 11)(10, 12),
+> (1, 4, 8)(2, 6, 10)(3, 7, 11)(5, 9, 12)]);;
+gap> mat := [[G.1, G.2], [G.1, G.1], [G.2, G.3]];;
+gap> S := ReesMatrixSemigroup(G, mat);;
+gap> R := Range(RMSNormalization(S));;
+gap> L := LeftTranslations(R);;
+gap> RT := RightTranslations(R);;
+gap> gpfunc := [G.1, G.2 * G.1];;
+gap> t := IdentityTransformation;;
+gap> l := LeftTranslationOfNormalRMS(L, gpfunc, t);
+<left translation on <simple semigroup of size 72, with 5 generators>>
+gap> r := RightTranslationOfNormalRMS(RT, gpfunc, t);
+Error, Semigroups: RightTranslationOfNormalRMS: 
+the second argument must be a list of group elements of length equal t\
+o the number of columns of the underlying semigroup of the first argum\
+ent,
+]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="BitranslationOfNormalRMS">
+  <ManSection>
+    <Func Name="BitranslationOfNormalRMS" Arg="H, l, r"/>
+    <Returns>A bitranslation.</Returns>
+    <Description>
+      <C>BitranslationOfNormalRMS</C>(<A>H, l, r</A>) functions the same as
+      <C>Bitranslation</C> but employs a more efficient method for determining
+      whether the two translations are linked, special to a normalised RMS.
+      <Example><![CDATA[
+gap> G := Group([(1, 2, 3, 5)(4, 10, 7, 12)(6, 11, 9, 8),
+> (1, 3)(2, 5)(4, 7)(6, 9)(8, 11)(10, 12),
+> (1, 4, 8)(2, 6, 10)(3, 7, 11)(5, 9, 12)]);;
+gap> mat := [[G.1, G.2], [G.1, G.1], [G.2, G.3]];;
+gap> S := ReesMatrixSemigroup(G, mat);;
+gap> R := Range(RMSNormalization(S));;
+gap> G := UnderlyingSemigroup(R);;
```

Comment:
```suggestion
```
You can delete this line because normalising a RMS/RZMS doesn't change its underlying (semi)group. So here `G` is still defined as it was a few lines above.